src/locale/locales/fr/messages.po

```diff
@@ -5853,6 +5890,10 @@ msgstr "Préférences des fils de discussion"
 msgid "Thread Preferences"
 msgstr "Préférences des fils de discussion"
 
+#: src/components/WhoCanReply.tsx:109
+msgid "Thread settings updated"
+msgstr "Mise à jour des paramètres du fil de discussion"
```

Comment:
Well, I think we want to inform the user that it did happen, so it should be this form:

```suggestion
msgstr "Paramètres du fil de discussion mis à jour"
```

Reply:
Oops, yes I should have rephrased this